### DISCUSSION TITLE
alarm panel integration? (Alarmo?)

### DISCUSSION THREAD

**Discussion by Menethoran** [Blackymas/NSPanel_HA_Blueprint#1779] [General]
Hey guys. Is there any way to get an Alarmo pin panel to show on our NSPanels through the blueprint? Ive dug in all the areas asnd cant seem to get a pin pad tro show on my panel. everything else works as intended.
any help would be appreciated

**Comment by Menethoran** [comment]
actually, it looks like issue 184 islikely going to push to fix this.

**Reply by edwardtfn** [reply]
Which model are you using? We are aware of an issue that is only on US model. That is already fixed in `dev` and will be included in the next release.

**Comment by edwardtfn** [comment]
Probably fixed by v4.2.5.
Please report as a bug if you still having issues.
